subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/fixture/GradleProfilerBuildExperimentRunner.java

```diff
@@ -57,12 +58,16 @@
     private final String jfrProfileTargetDir;
     private final ProfilerFlameGraphGenerator flameGraphGenerator;
     private final BenchmarkResultCollector resultCollector;
+    private final Profiler profiler;
 
     public GradleProfilerBuildExperimentRunner(BenchmarkResultCollector resultCollector) {
         this.jfrProfileTargetDir = org.gradle.performance.fixture.Profiler.getJfrProfileTargetDir();
```

Comment:
I think this can be a local variable now.

Reply:
Good catch, converted!